pkg/integration/tests/file/copy_menu.go

```diff
@@ -0,0 +1,153 @@
+package file
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/config"
+	. "github.com/jesseduffield/lazygit/pkg/integration/components"
+)
+
+// note: this is required to simulate the clipboard during CI
+func expectClipboard(t *TestDriver, matcher *TextMatcher) {
+	defer t.Shell().DeleteFile("clipboard")
+
+	t.FileSystem().FileContent("clipboard", matcher)
+}
+
+var CopyMenu = NewIntegrationTest(NewIntegrationTestArgs{
+	Description:  "The copy menu allows to copy name and diff of selected/all files",
+	ExtraCmdArgs: []string{},
+	Skip:         false,
+	SetupConfig: func(config *config.AppConfig) {
+		config.UserConfig.OS.CopyToClipboardCmd = "echo {{text}} > clipboard"
+	}, SetupRepo: func(shell *Shell) {}, Run: func(t *TestDriver, keys config.KeybindingConfig) {
```

Comment:
Please put the `Run` member on a newline, it's easier to parse.

Reply:
Yeah, totally. I probably messed this up while changing the test. Thanks.